hid-xpadneo/src/udev_rules/99-xpadneo.rules

```diff
@@ -22,3 +22,8 @@ ACTION=="add", \
 KERNEL=="0005:045E:02FD.*|0005:045E:02E0.*", \
 SUBSYSTEM=="hid", \
 RUN:="/bin/sh -c 'echo xpadneo udev: $kernel > /dev/kmsg; modprobe hid_xpadneo; echo $kernel > /sys/bus/hid/drivers/hid-generic/unbind; echo $kernel > /sys/bus/hid/drivers/microsoft/unbind; echo $kernel > /sys/bus/hid/drivers/xpadneo/bind; echo xpadneo udev: ok > /dev/kmsg'"
+
+ACTION=="add", \
+KERNEL=="0005:045E:0B05.*|0005:045E:0B05.*", \
```

Comment:
I believe there is no need for double add action. You can extend hardware addresses in 1st rule

Reply:
I think so too, could you please change that?

Reply:
Done :)